docs/install.md

```diff
@@ -14,147 +14,205 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
-# Installation
+# Recommended Installation Method
 
-There are 4 ways to use Triton Model Analyzer:
+<br>
 
-## Building the Dockerfile
+## Triton SDK Container with Docker Launch Mode
 
-The recommended way to use Model Analyzer is by building the Model Analyzer's
-docker yourself. This installation method uses `--triton-launch-mode=local` by
-default, as all the dependencies will be available. First, clone the Model
-Analyzer's git repository, then build the docker image.
+---
 
-```
-$ git clone https://github.com/triton-inference-server/model_analyzer.git -b <rXX.YY>
+The recommended way to install Model Analyzer is from the Triton SDK docker
+container available on the [NVIDIA GPU Cloud
+Catalog](https://ngc.nvidia.com/catalog/containers/nvidia:tritonserver).<br><br>
+
+**1. Choose a release version**  
+The format is `<rYY.MM>` where, `YY`=year and `MM`=month
 
-$ cd ./model_analyzer
+For example, the release from May of 2022 is `<r22.05>`
```

Comment:
the end of May
Also, I think we need to remove the <> in some places. When I read this I think I'm supposed to add the <>.

Reply:
Fixed